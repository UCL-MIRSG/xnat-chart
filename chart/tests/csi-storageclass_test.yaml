templates:
  - templates/csi-storageclass.yaml

tests:
  - it: csi-storageclass manifest should match snapshot
    set:
      csiStorageClasses.smb.provisioner: smb.csi.k8s.io
      csiStorageClasses.smb.source: //server/share
      csiStorageClasses.smb.provisionerSecretName: smbcreds
      csiStorageClasses.smb.provisionerSecretNamespace: smb
      csiStorageClasses.smb.nodeStageSecretName: smbcreds
      csiStorageClasses.smb.nodeStageSecretNamespace: smb
    asserts:
      - matchSnapshot: {}
