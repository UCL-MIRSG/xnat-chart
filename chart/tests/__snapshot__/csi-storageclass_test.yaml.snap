csi-storageclass manifest should match snapshot:
  1: |
    allowVolumeExpansion: true
    apiVersion: storage.k8s.io/v1
    kind: StorageClass
    metadata:
      name: smb
    mountOptions:
      - dir_mode=0777
      - file_mode=0777
    parameters:
      csi.storage.k8s.io/node-stage-secret-name: smbcreds
      csi.storage.k8s.io/node-stage-secret-namespace: smb
      csi.storage.k8s.io/provisioner-secret-name: smbcreds
      csi.storage.k8s.io/provisioner-secret-namespace: smb
      source: //server/share
    provisioner: smb.csi.k8s.io
    reclaimPolicy: Delete
    volumeBindingMode: Immediate
